lidarTopic: "/velodyne_points"               # Point cloud data
imuTopic: "/nav/filtered_imu/data"                         # IMU data
bagPath: "/home/daniel/Documents/output.bag"                         # IMU data
extrinsicsPitch: 0
extrinsicsRoll: 0
extrinsicsYaw: 90
